app/(main)/components/Features.tsx

```
@@ -457,11 +424,129 @@ export function HomepageFeatures({ data }: { data: FetchedData }) {
+                  </li>
+                  <li>
+                    <Text variant="p2">Seal your tokens and continue to collect rewards</Text>
+                  </li>
+                </List>
+                <br />
+                Your MKR tokens, as well as any rewards you accumulate, are supplied to a non-custodial smart
+                contract, which means no intermediary has custody of those tokens. With Sky, you always remain
+                in control of your funds.
+              </Text>
+            </>
+          }
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)